build.gradle

```diff
@@ -225,47 +218,40 @@ def ossrhUsername = hasProperty('ossrhUsername') ? property('ossrhUsername') : '
 def ossrhPassword = hasProperty('ossrhPassword') ? property('ossrhPassword') : ''
 
 publishing {
-
     publications {
-        // Uncomment below if you want to run "publishMavenLocal"
-        maven(MavenPublication) {
-            pom.withXml {
-                asNode().with {
-                    appendNode('packaging', 'jar')
-                    appendNode('name', 'cadence-client')
-                    appendNode('description', description)
-                    appendNode('url', 'https://github.com/uber-java/cadence-client')
-                    appendNode('scm').with {
-                        appendNode('url', 'https://github.com/uber-java/cadence-client')
-                        appendNode('connection', 'git@github.com:uber-java/cadence-client.git')
+        mavenJava(MavenPublication) {
+            from components.java
+            artifact javadocJar
+            artifact sourcesJar
+            pom {
+                name = 'cadence-client'
+                description = 'Uber Cadence Java Client'
+                url = 'https://github.com/uber-java/cadence-client'
+                scm {
+                    url = 'https://github.com/uber-java/cadence-client'
+                    connection = 'scm:git:git@github.com:uber-java/cadence-client.git/'
+                    developerConnection = 'scm:git:git@github.com:uber-java/cadence-client.git/'
+                }
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
                     }
-                    appendNode('licenses').with {
-                        appendNode('license').with {
-                            appendNode('name', 'The Apache License, Version 2.0')
-                            appendNode('url', 'http://www.apache.org/licenses/LICENSE-2.0.txt')
-                        }
+                }
+                developers {
+                    developer {
```

Comment:
Maybe there's a group cadence alias we can point to instead?

Reply:
I think we can add mailingLists. I'll try to release first and in another PR to add it